client/my-sites/plans/jetpack-plans/jetpack-complete-page/components/cta-buttons/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { PLAN_JETPACK_COMPLETE } from '@automattic/calypso-products';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import Button from 'calypso/components/forms/form-button';
+import getJetpackAdminUrl from 'calypso/state/sites/selectors/get-jetpack-admin-url';
+import { getSelectedSiteSlug, getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { buildCheckoutURL } from '../../../get-purchase-url-callback';
+
+import './style.scss';
+
+const CtaButtons = () => {
+	const translate = useTranslate();
+	const siteSlug = useSelector( getSelectedSiteSlug );
+	const siteId = useSelector( getSelectedSiteId );
+
+	const checkoutURL = siteSlug
+		? buildCheckoutURL( siteSlug, PLAN_JETPACK_COMPLETE )
+		: `/checkout/${ PLAN_JETPACK_COMPLETE }`;
+
+	const stayFreeURL = useSelector(
+		( state ) => getJetpackAdminUrl( state, siteId ) ?? 'https://jetpack.com/'
```

Comment:
Since the page exists and is specific, I've changed it to that one.  We can always address that page later either ourselves with some help with marketing or maybe MarTech can take it on at some point.